README.md

```diff
@@ -225,10 +225,114 @@ that they appear in `builder.toml`. This image's base will be the build image as
 > It's important to note that the buildpacks in a builder are not actually executed until
 > [`build`](#building-explained) is run.
 
+#### Lifecycle
+
+A builder also includes a layer with a series of binaries that are executed during [`build`](#building-explained).
+These binaries represent a specific implementation of the buildpack **lifecycle**, as defined by the
+[Cloud Native Buildpacks API Specification](https://github.com/buildpack/spec#buildpack-api-v3---specification). While
+most users will find the [default lifecycle implementation](https://github.com/buildpack/lifecycle#lifecycle)
+sufficient, users are free to implement their own. See
```

Comment:
`pack` does rely on unspecified flags that are provided by the default lifecycle. So there is no guarantee that any given lifecycle is compatible with pack, even if it complies with the spec (Another way to put it: this particular platform has a contract with the lifecycle that is a superset of the specified contract). Perhaps we shouldn't mention custom lifecycle implementations in the `pack` docs.

Reply:
Great point. I agree we should change the wording. It's updated now.

Reply:
The previous wording was perhaps too much detail anyway, so I like the way it is now.